androidapp/HopSquadApp/app/src/main/java/com/hopsquad/hopsquadapp/api/BeerRepository.java

```diff
@@ -0,0 +1,54 @@
+package com.hopsquad.hopsquadapp.api;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import java.util.List;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+/**
+ * Created by memo on 15/11/17.
+ */
+
+public class BeerRepository {
+
+    // This is for debugging purposes on the simulator
+    // TODO: switch to actual project on Google Cloud
+    public static final String BASE_URL = "http://10.0.2.2:8080/";
```

Comment:
The app has some values ready in the api, should we update this now?

Reply:
I'll add that in the new branch `android-orders`.